src/main/java/com/cag/cagbackendapi/dtos/ProfileDto.kt

```diff
@@ -0,0 +1,23 @@
+package com.cag.cagbackendapi.dtos
+
+import com.cag.cagbackendapi.entities.UserEntity
+import java.util.*
+
+data class ProfileDto(
+        var profile_id: UUID?,
+        var pronouns: String?,
+        var lgbtqplus_member: Boolean?,
+        var gender_identity: String?,
+        var comfortable_playing_man: Boolean?,
+        var comfortable_playing_women: Boolean?,
+        var comfortable_playing_neither: Boolean?,
+        var comfortable_playing_transition: Boolean?,
+        var height_inches: Int?,
+        var agency: String?,
+        var website_link_one: String?,
+        var website_link_two: String?,
+        var website_type_one: String?,
+        var website_type_two: String?,
+        var bio: String?,
+        var user: UserEntity? = null)
```

Comment:
Can you try and change this to UserDto and see if everything still works? We should try to not mix our DTOs and our Entities together